pandas/core/dtypes/cast.py

```diff
@@ -439,6 +444,59 @@ def changeit():
     return result, False
 
 
+def maybe_casted_values(index, codes=None):
+    """
+    Convert an index, given directly or as a pair (level, code), to a 1D array.
+    Parameters
+    ----------
+    index : Index
+    codes : sequence of integers (optional)
+    Returns
```

Comment:
pls format this doc-string like normal (e.g. a blank line before Parameters, and Returns)

Reply:
Done